src/components/policy/policy_external/test/update_status_manager_test.cc

```diff
@@ -72,18 +73,53 @@ class UpdateStatusManagerTest : public ::testing::Test {
   void TearDown() OVERRIDE {}
 };
 
-// TODO(AKutsan) : APPLINK-31222 use miliseconds as parameter of OnUpdateSentOut
-// function
-TEST_F(
-    UpdateStatusManagerTest,
-    DISABLED_OnUpdateSentOut_WaitForTimeoutExpired_ExpectStatusUpdateNeeded) {
+namespace {
+class WaitAsync {
```

Comment:
@LitvinenkoIra please split commits not by files, but by logic. One commit - checne secconsto miliseconds, second one - creatin WaitAsyc class and using it